web/src/components/FilingsCalendar.tsx

```diff
@@ -59,16 +62,24 @@ export const FilingsCalendar = (props: Props): ReactElement => {
       return <></>;
     }
 
+    const sortedFilteredOperatingFilings = groupBy(
```

Comment:
I wonder if we could have a more descriptive variable name here. I think the groupby is confusing (I'm glad you pulled it out and named it so that we have the opportunity to clear up that confusion and name it something descriptive)